### DISCUSSION TITLE
Text rendering on dark backgrounds seems bolder than other applications

### DISCUSSION THREAD

**Discussion by ornamentist** [zed-industries/zed#6567] [Feature Requests]
I may not have used the Zed settings correctly here, but text rendered on a dark background seems much heavier or bolder than other applications.  With Roboto Mono Thin, here's Zed:

<img width="672" alt="Screenshot 2023-04-25 at 8 50 58 am" src="https://user-images.githubusercontent.com/211742/234133211-4f5a3e59-030b-4523-a717-7d640aac2920.png">

Then VSCode:

<img width="641" alt="Screenshot 2023-04-25 at 8 48 10 am" src="https://user-images.githubusercontent.com/211742/234133288-91aa10c5-a963-4149-b1e9-1d86dd6a682a.png">

Then ITerm2:

<img width="693" alt="Screenshot 2023-04-25 at 8 49 02 am" src="https://user-images.githubusercontent.com/211742/234133324-15ded2f9-8e57-4520-964f-4a28588244b6.png">

<img width="394" alt="Screenshot 2023-04-25 at 8 49 44 am" src="https://user-images.githubusercontent.com/211742/234133342-0d91a27c-c307-41f3-8cbf-74e5b4cae7c5.png">

We may need a setting for font anti-aliasing or weight?

**Comment by mattly** [comment]
it actually seems to me that Zed doesn't yet support font family changes -- if you notice, in your screenshot it's not actually rendering text in Roboto, rather their font, Zed Mono. I've tried changing the settings to a number of different fonts and it never seems to "take"

**Reply by ornamentist** [reply]
Good catch!

**Reply by ojkelly** [reply]
Hmm, I've been using a different font for quite a while now and it still works.

I'm using the OTF version of Berkeley Mono.

Is it possible you're using a TTF or similar and Zed doesn't read it?

Zed Mono
<img width="601" alt="zed-mono" src="https://github.com/zed-industries/community/assets/666613/205f23ad-5464-4b42-b7a4-e1cd5c9b5df0">

Berkeley Mono
<img width="575" alt="zed-berkeley" src="https://github.com/zed-industries/community/assets/666613/7e5439ff-de5c-4a24-bca9-6795af370d19">

Look at the lowercase `t` on line 12 if you can't tell them apart.

**Comment by iamnbutler** [comment]
It is likely you are not using the correct format for the name of font. This can differ from font to font, try consulting the "Identifiers" section in the sidebar in FontBook to find the correct name. 

For example with Arial Black: 
<img width="1728" alt="CleanShot 2023-06-29 at 12 00 26@2x" src="https://github.com/zed-industries/community/assets/1714999/4343526e-f169-4512-be8d-5a809f64d862">

I did try Roboto Mono however, it appears to not apply which may mean whatever format it is in is not supported. Try another font, or roboto in another format and see if that works for you.

**Reply by ornamentist** [reply]
I’ll look for an OTF version of Roboto Mono. In the meantime would it be a bad idea to request TTF font format support?

**Comment by mattly** [comment]
yeah, my buffer font setting started magically working after the most recent update
